src/types/LegendColorPair.ts

```diff
@@ -0,0 +1,4 @@
+export interface LegendColorPair {
```

Comment:
`LegendColorPair` -> `ILegendColorPair`

TS convention is to start all interfaces with an `I`